### ISSUE TITLE
[Bug Report] Tab management broken in normal mode | Cannot close specific tabs, Tab text partially hidden, Close button hidden, Multiple tabs hidden

### ISSUE THREAD

**Issue by ProHackTech** [microsoft/terminal#2223]
# Environment

```none
Windows Version: Windows 10 Education 1903
OS Build: 18362.10005
Windows Terminal version: 0.3.2142.0


Any other software?
```

# Steps to reproduce

Opening more than certain tabs breaks the tab management. There are several issues faced:

- Multiple tabs become hidden and there is no way to know how many tabs are open and way to access them.

- Tab text/label is partially hidden along with the close button for each tab after the first tab.

Unless the window is maximized, there is no way to even find the new tabs opened. The configuration used is default.

To reproduce:

1. Open Windows Terminal

2. Open many tabs

The following GIF demonstrates the issue:

<img src="https://media.giphy.com/media/ZEfkAbM6HdNflKf7R1/giphy.gif" alt="Broken Tab Management">

# Expected behavior

- Tab text/label and close button is not hidden

- There is a way to know the number of tabs open and a way to access each one

# Actual behavior

- Multiple tabs become hidden and there is no way to know how many tabs are open and way to access them.

- Tab text/label is partially hidden along with the close button for each tab after the first tab.

GIF image above showcases the bug.

**Comment by TayYuanGeng** [comment]
This is similar to #1862 and will be fixed by #1205

**Comment by DHowett-MSFT** [comment]
Also, #1375 #444 #597
